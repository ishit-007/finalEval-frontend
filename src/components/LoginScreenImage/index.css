.loginImage {
  position: relative;
}
.loginImage img {
  height: 800px;
  width: 800px;
}
.loginImage .content {
  position: absolute;
  color: #4f4f4f;
  transform: translate(250px,-790px);
}
.loginImage .content h1{
  padding-top: 30px;
}
