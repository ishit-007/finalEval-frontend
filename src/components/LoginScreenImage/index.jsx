import LoginScreenImage from '../LoginScreenImage';
import { render } from '@testing-library/react';
import React from 'react';

describe('LoginScreenImage', () => {
  it('renders the component', () => {
    const { getByAltText, getByText } = render(<LoginScreenImage />);
    const image = getByAltText('Login Screen');
    const title1 = getByText(/Design API's fast,/i);
    const title2 = getByText(/Manage content easily./i);
    expect(image).toBeInTheDocument();
    expect(title1).toBeInTheDocument();
    expect(title2).toBeInTheDocument();
  });
});
