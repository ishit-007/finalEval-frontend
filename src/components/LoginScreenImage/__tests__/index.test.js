import LoginScreenImage from '..';
import renderer from 'react-test-renderer';
import React from 'react';

describe('LoginScreenImage', () => {
  it('should render the Image component', async () => {
    const tree = renderer.create(<LoginScreenImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});