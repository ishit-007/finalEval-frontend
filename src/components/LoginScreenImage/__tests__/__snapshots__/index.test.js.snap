// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreenImage should render the Image component 1`] = `
<div
  className="loginImage"
>
  <img
    alt="Login Screen"
    src="undraw-upload-re-pasx@3x.png"
  />
  <div
    className="content"
  >
    <h1>
      Design API's fast,
    </h1>
    <h1>
      Manage content easily.
    </h1>
  </div>
</div>
`;
